src/main/webapp/resources/js/components/samples/components/SampleAnalyses.jsx

```diff
@@ -1,43 +1,153 @@
 import React from "react";
-import { Input, Space, Table } from "antd";
+import { Col, Input, Row, Table, Typography } from "antd";
+import { useDispatch, useSelector } from "react-redux";
+import { fetchSampleAnalyses } from "../../../apis/samples/samples";
+import { setSampleAnalyses } from "../sampleAnalysesSlice";
+
+import { setBaseUrl } from "../../../utilities/url-utilities";
+import { formatInternationalizedDateTime } from "../../../utilities/date-utilities";
+import {
+  IconCheckCircle,
+  IconClock,
+  IconCloseCircle,
+  IconLoading,
+} from "../../icons/Icons";
+import { blue6, green6, grey6, red6 } from "../../../styles/colors";
 
 const { Search } = Input;
+const { Paragraph } = Typography;
+const commonIconStyle = {
+  fontSize: "16px",
+};
+
 /**
  * React component to display sample analyses
  *
  * @returns {JSX.Element}
  * @constructor
  */
 export function SampleAnalyses() {
+  const [filteredSubmissions, setFilteredSubmissions] = React.useState(null);
+  const dispatch = useDispatch();
+
+  const { sample } = useSelector((state) => state.sampleReducer);
+
+  const { analyses, loading } = useSelector(
+    (state) => state.sampleAnalysesReducer
+  );
+
+  /*
+  On page load get the sample analyses
+   */
+  React.useEffect(() => {
+    fetchSampleAnalyses({ sampleId: sample.identifier })
+      .then((analysesList) => {
+        dispatch(setSampleAnalyses({ analyses: analysesList }));
+      })
+      .catch((error) => {});
+  }, []);
+
   // Columns for the table
   const columns = [
     {
-      title: i18n("SampleAnalyses.sampleId"),
-      key: "sample_id",
+      title: i18n("SampleAnalyses.analysisSubmissionName"),
+      dataIndex: "name",
+      key: "name",
+      render(name, data) {
+        return (
+          <Paragraph ellipsis={{ rows: 1 }}>
```

Comment:
This actually works with children? I thought the text had to be a direct child.  Test it out, if it does not work, you might be able to next the paragraph inside the anchor tag

Reply:
Removed paragraph and set ellipsis: true and width for column in 84b01a29dc2ced20afe8e58598ff464cf93399bb